WordPress/gutenberg
# Add LINE to social icons block #57830

## What?
The PR is adding a new LINE icon to the social icons block, addressing the need to include LINE messenger, which is popular and essential in some Asian countries. The LINE icon is introduced to complement the existing social icons.

## Why?
This PR is necessary to enhance the social media integration of the application. The addition of the LINE icon caters to the user base in Asian countries where LINE is widely used. It aligns with the goal of providing a comprehensive set of social icons for users to choose from (Issue: "Add LINE to social icons block").


## How?
The implementation involves introducing a new SVG component specifically for the LINE icon. The code structure is modeled after existing social icons, ensuring consistency. The SVG code for the LINE icon is generated based on a provided sample (uploaded SVG) to accurately represent the LINE logo.

## Testing Instructions
1. Open a post or page.
2. Insert a social icons block.
3. Observe the presence of the new LINE icon in the social icons list.
4. Click on the LINE icon to ensure it functions as expected.
5. Test the responsiveness of the LINE icon in various screen sizes.
6. Save the changes and preview the post/page to verify the LINE icon persists.
